cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5447,7 +5447,7 @@
 
   <property>
     <name>feature.lifecycle.management.edit.enabled</name>
-    <value>false</value>
+    <value>true</value>
```

Comment:
Should we make it final?

Reply:
nothing in cdap-default should be final. At that point there is no use in having a config setting, it is the same as having a constant in the code.